install/kubernetes/helm/istio/values-istio-example-sds-token-vault.yaml

```diff
@@ -0,0 +1,40 @@
+global:
```

Comment:
why create a new file rather than uses [existing one for vault example](https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml) ?

Reply:
To avoid break the user guide that uses the existing one.

Reply:
so after this PR get in, you will update user guide to point to this file, then remove previous one https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml ?

Reply:
Yes.